psi4/driver/driver_cbs.py

```diff
@@ -917,135 +1137,468 @@ def return_energy_components():
         for mplevel2 in range(2, mplevel):
             VARH[f'mp{mplevel}'][f'mp{mplevel2}'] = f'MP{mplevel2} TOTAL ENERGY'
 
-    # Integrate CFOUR methods
+    # Incorporate CFOUR methods
     VARH.update(cfour_psivar_list())
+
+    # Incorporate density functionals
+    # Somewhat confusingly, we need both {funcname} and {funcname}-dft, even 
+    # though they point at the same energy:
+    #   - {funcname} matches the name that is extracted as a wfn, either by
+    #     specifications in the metadata directly, or from a method/basis format
+    #   - {funcname}-dft is the component that corresponds to the total energy
+    #     of the functional recipe, useful for "naive" extrapolations
+    # The other components -fctl, -disp, -dh, and -nl are self-explanatory,
+    # however note that currently -fctl includes -nl but does not include -disp
+    for funcname in functionals:
+        if funcname == "hf" or funcname == "scf":
```

Comment:
```suggestion
        if funcname in {"hf", "scf"}:
```

Nitpicky, but might as well.